src/webhook/mutation/pod_mutator/pod_mutator_test.go

```diff
@@ -90,11 +90,18 @@ func TestHandlePodMutation(t *testing.T) {
 		require.NoError(t, err)
 		assert.NotNil(t, mutationRequest.InstallContainer)
 		assert.Len(t, mutationRequest.Pod.Spec.InitContainers, 2)
+
+		require.NotNil(t, *mutationRequest.Pod.Spec.InitContainers[1].SecurityContext.Privileged)
```

Comment:
isn't `NotNil` kind of implied if the value is `False`? Could just be `require.False`

Reply:
there is an error here but its that I should have written `require.NotNil(t, mutationRequest....`

you have to do this because if you just `require.False(t, *mutationRequest....` then it can fail with a panic, as you tried to get the value behind a pointer that is `nil`, so you should check that the pointer itself must not be nil and check its value after